src/components/field-label/field-label.js

```diff
@@ -9,66 +10,67 @@ import Label from '../label';
 import Constraints from '../constraints';
 import Spacings from '../spacings';
 
-export const FieldLabel = props => (
-  <Constraints.Horizontal constraint={props.horizontalConstraint}>
-    <Spacings.Stack scale="xs">
-      <Spacings.Inline alignItems="flexStart" scale="xs">
-        <Text.Wrap>
-          <Label
-            isBold={true}
-            isRequiredIndicatorVisible={props.hasRequiredIndicator}
-            htmlFor={props.htmlFor}
-          >
-            {props.title}
-          </Label>
-        </Text.Wrap>
-        {props.onInfoButtonClick && (
-          <IconButton
-            label="More Info"
-            icon={<InformationIcon size="medium" />}
-            size="small"
-            onClick={props.onInfoButtonClick}
-          />
-        )}
-      </Spacings.Inline>
-
-      {props.hint && (
-        <Spacings.Inline alignItems="center" scale="s">
-          {props.hintIcon && (
-            <div
-              css={css`
-                display: flex;
-              `}
+export const FieldLabel = props => {
+  if (props.hintIcon && props.hintIcon.props && props.hintIcon.props.size) {
```

Comment:
if this is the desired behaviour, we should warn when users try to set the size.